RELEASING.md

```diff
@@ -36,21 +38,78 @@
 
-1. _Commit_ and _push_ any change from the
-   [repository in Weblate](https://translations.metabrainz.org/projects/musicbrainz/#repository),
-   and update your local `translations` branch.
+1. From the [repository in Weblate](https://translations.metabrainz.org/projects/musicbrainz/#repository):
+
+   1. If there are any “Pending changes”, “Commit” those and wait a bit before reloading the page.
+
+   2. If there are any commits “missing in the push branch”, “Push” those and wait (some more time) before reloading the page.
 
-2. Merge `translations` to `beta` (`git merge --log=876423 --no-ff translations`) and push.
+   3. If there are any “missing commits” from “upstream” (beta branch), “Update” the “Weblate repository” and wait (even longer) before reloading the page.
+
+2. Update your local `translations` branch and merge it to `beta` and push as follows:
+   ```sh
+   git fetch origin && \
+   git checkout origin/translations -B translations && \
+   git checkout beta && \
+   git merge --log=876423 --no-ff translations && \
+   git push
+   ```
    Wait until [CircleCI](https://circleci.com/gh/metabrainz/musicbrainz-server) is happy
    with this merge as some unmatching translations can break building Docker images.
 
+### Draft blog post
+
+[Create a new draft from the template](https://wordpress.com/post/blog.metabrainz.org?jetpack-copy=8634).
+The list of tickets can be copied from the auto-generated release notes,
+by going to the [Releases](http://tickets.musicbrainz.org/browse/MBS#selectedTab=com.atlassian.jira.plugin.system.project%3Aversions-panel) tab,
+selecting the version you are going to release,
+and clicking “Release Notes” in the top.
+
+If there are any React conversion tasks
+(which should be sub-tasks of [MBS-8609](https://tickets.metabrainz.org/browse/MBS-8609)),
+move these to the section “React Conversion Task” at the end,
+and move other (sub-)tasks under the section “Other Task”.
+Otherwise, just put all the tasks under a unique section “Task”.
+
+Last but not least, thank (in order of rarity):
+* code contributors, who can be listed with `./script/list_code_contributors`,
+* translators, who can be listed with `./po/list_translators`,
+* reporters of each addressed issue at least,
+  and every other constructive feedback providers if possible:
+  * reporters of [tickets addressed in beta](https://tickets.metabrainz.org/issues/?filter=10715&jql=project%20%3D%20MBS%20AND%20status%20%3D%20%22In%20Beta%20Testing%22%20ORDER%20BY%20reporter%20ASC%2C%20type%2C%20key)
+  * reporters of [tickets about beta](https://tickets.metabrainz.org/issues/?jql=project%20%3D%20MBS%20AND%20%28summary%20~%20Beta%20OR%20fixVersion%20%3D%20Beta%29%20AND%20created%20%3E%3D%20-2weeks%20ORDER%20BY%20reporter%20ASC) (assuming a two weeks cycle)
+  * comment authors (using the same lists ordered by “Watchers” to find the most popular tickets, see also pull requests)
```

Comment:
Thanking comment authors seems like it would look slightly ridiculous to me to be honest (unless it is very specific ones that were particularly useful).

Reply:
I did that a few times before, when their contribution is thorough enough, as sometimes comments go more into details than the tickets. “constructive feedback” was already mentioned above, but I can add “thorough” here too, and make it more optional because it can be tricky to remember them all:
```suggestion
  * thorough comment authors if possible (using the same lists ordered by “Watchers” to find the most popular tickets, see also pull requests)
```

Reply:
Ok, if you mean "especially good commenters", that seems sensible to me, your amendment seems sensible.